src/DwitTech.AccountService.Core/Extensions/ServiceCollectionExtension.cs

```diff
@@ -68,15 +70,10 @@ public static void ConfigureAuthentication(this IServiceCollection services, ICo
                     ValidateIssuerSigningKey = true,
                     ValidateIssuer = true,
                     ValidateLifetime = true,
-                    ValidateAudience = true,
-                    ValidIssuer = configuration["Jwt:Issuer"],
+                    ValidateAudience = false,
                     ValidAudiences = new List<string> { configuration["JWT:Audience"] },
-                    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(configuration["Jwt:Key"]))
```

Comment:
These changes were not intentional. I assume it was because of the stash and pop I did. I'll correct it.